operator/v1/types_clustermanager.go

```diff
@@ -243,6 +254,33 @@ type HostedClusterManagerConfiguration struct {
 	WorkWebhookConfiguration WebhookConfiguration `json:"workWebhookConfiguration,omitempty"`
 }
 
+// WebhookDefaultConfiguration represents configuration for webhooks running in "Default" mode in the hub cluster
+type WebhookDefaultConfiguration struct {
+	// Port represents the port of a webhook-server. The default value of Port is 9443.
+	// +optional
+	// +kubebuilder:default=9443
+	// +kubebuilder:validation:Maximum=65535
+	Port int32 `json:"port,omitempty"`
+
+	// HealthProbeBindAddress represents the healthcheck address of a webhook-server. The default value is ":8000".
+	// Healthchecks may be disabled by setting a value of "0" or "".
+	// +optional
+	// +kubebuilder:default=":8000"
+	HealthProbeBindAddress string `json:"healthProbeBindAddress"`
+
+	// MetricsBindAddress represents the metrics address of a webhook-server. The default value is ":8080"
+	// Metrics may be disabled by setting a value of "0" or "".
+	// +optional
+	// +kubebuilder:default=":8080"
+	MetricsBindAddress string `json:"metricsBindAddress"`
+
+	// HostNetwork enables running webhook pods with hostNetwork: true
+	// This may be required in some installations, such as EKS with Calico CNI,
+	// to allow the API Server to communicate with the webhook pods.
+	// +optional
+	HostNetwork bool `json:"hostNetwork,omitempty"`
```

Comment:
we do not set a boolean field commonly. It could be a enum string which is more extensible.

Reply:
I see, we will use this to directly set to webhook pod. That is ok in this case.

Reply:
Ok great. Thanks for letting me know about the preference for enum, I will keep that in mind in future